Wallet connect 2 (#40)
Psychedelic/plug-docs: docs/getting-started/alpha-wallet-connect.md

The alpha Plug Mobile auth comes with 3 packages: 

> - [**Alpha Android APK**](https://play.google.com/apps/testing/co.psychedelic.plug) - the updated Android application that can talk using Wallet Connect.

Reviewer: ```suggestion
- [**Alpha Android APK**](https://play.google.com/apps/testing/co.psychedelic.plug) - the updated Android App that can talk using Wallet Connect.
```

> - [**Alpha iOS APP**](https://testflight.apple.com/join/yID6TEpV) - the updated iOS application that can talk using Wallet Connect.

Reviewer: ```suggestion
- [**Alpha iOS APP**](https://testflight.apple.com/join/yID6TEpV) - the updated iOS application that can communicate using Wallet Connect.
```

Once you're ready to test the implementation, you can: 

1. [Install the Alpha Android APK](https://play.google.com/apps/testing/co.psychedelic.plug) in a phone and go to the dapp in the browser. The name of the updated application should be `Plug - Crypto Wallet (Beta)`.

> 1. [Install the Alpha iOS APP](https://testflight.apple.com/join/yID6TEpV) in a phone and go to the dapp in the browser. The name of the updated application should be `Plug - Crypto Wallet (Beta)`.

Reviewer: ```suggestion
1. [Install the Alpha iOS APP](https://testflight.apple.com/join/yID6TEpV) in a phone and go to the dapp in the browser. The name of the updated application should be `Plug - Crypto Wallet [v0.3.0(7)]`.
```

> 2. Connect to the application using the Plug Alpha Android or iOS app.

Reviewer: ```suggestion
2. Connect to the application using the Plug Alpha Android or iOS App.
```
